### ISSUE TITLE
Share buttons not accessible for keyboard navigation

### ISSUE THREAD

**Issue by AndreasJacobsen** [elementor/elementor#8615]
## Prerequisites

- [X ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X ] The issue still exists against the latest stable version of Elementor.

## Description

It is not possible to tab into individual share buttons for blogposts, these are the default Elementor share buttons. I have found no way to fix this 
![image](https://user-images.githubusercontent.com/8527173/61535075-9a476380-aa31-11e9-916e-5a35bf0cc21a.png)

## Steps to reproduce
Add the "Share buttons" elementor element to your blog posts.

## Isolating the problem

- [X ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X ] This bug happens with a default WordPress theme active.
- [ X] I can reproduce this bug consistently using the steps above.

## Extra details
The national law in the country I work from (Norway) requires WCAG 2.0 A and AA compliance for all web pages. I am unable to active wcag compliance with the social media sharing buttons currently active in Elementor v2.6.5 with Elementor Pro v2.5.14. WCAG compliance and accessible website is also important where not required by law. 

Users should have the ability to tab into each individual button,  the buttons also need a role=button and the ability to add aria-label would also be nice. Currently the buttons are only div's

**Comment by rami-elementor** [comment]
Duplicate of #11299

**Comment by rami-elementor** [comment]
Ok, this issue has been resolved. The next Elementor Pro patch will include a fix.

Thanks!

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v3.6.3

Feel free to update

Thanks!
